defaults/main.yml

```diff
@@ -552,7 +548,7 @@ maas_pip_packages:
   - python-keystoneclient
   - python-magnumclient
   - python-neutronclient
-  - python-novaclient<7.0.0
+  - python-novaclient
```

Comment:
We should absolutely care! IMO we should put in some effort into converting these openstack service clients to use the openstacksdk. Similar to what is done here: https://github.com/rcbops/rpc-maas/pull/247

However, this isn't necessary for the issue-at-hand.